code/modules/jobs/job_types/serfs/gaffer.dm

```diff
@@ -11,10 +11,10 @@
 	//so it fits better with the wild cards that is this demographic of people
 	//having said that I am gate keeping the moment felinids are in the damn game
 	allowed_ages = list(AGE_MIDDLEAGED,AGE_OLD, AGE_IMMORTAL) //AGE_OLD with the ring on? I say unlikely - clown
-	tutorial = "Forced out of your old adventure party, you applied to the Mercenary guild. Eventually becoming\
+	tutorial = "Forced out of your old adventure party due to monetary issues, you applied to the Mercenary guild. Eventually becoming \
```

Comment:
```suggestion
	tutorial = "Forced out of your old adventure party, you applied to the Mercenary guild. Eventually becoming \
```
Like clown said, its better left open ended like similar things. And I say we don't ignore the say of the one who created gaffer in the first place, if I were him I would feel bad.

There are multiple things in lore that are known, but left ambigious for player interpretation.